<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MacroControl</class>
 <widget class="QGroupBox" name="MacroControl">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>106</width>
    <height>118</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>GroupBox</string>
  </property>
  <property name="title">
   <string>Macros</string>
  </property>
  <property name="alignment">
   <set>Qt::AlignCenter</set>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <property name="margin">
    <number>1</number>
   </property>
   <property name="spacing">
    <number>1</number>
   </property>
   <item row="0" column="0">
    <widget class="QScrollArea" name="scrollArea">
     <property name="widgetResizable">
      <bool>true</bool>
     </property>
     <widget class="QWidget" name="area">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>96</width>
        <height>71</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" name="arrangeButtons">
       <property name="spacing">
        <number>1</number>
       </property>
       <property name="margin">
        <number>1</number>
       </property>
      </layout>
     </widget>
    </widget>
   </item>
   <item row="1" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout">
     <property name="spacing">
      <number>1</number>
     </property>
     <item>
      <widget class="QRadioButton" name="lang0">
       <property name="text">
        <string>B</string>
       </property>
       <property name="checked">
        <bool>true</bool>
       </property>
       <attribute name="buttonGroup">
        <string notr="true">languageBox</string>
       </attribute>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="lang1">
       <property name="text">
        <string>e</string>
       </property>
       <attribute name="buttonGroup">
        <string notr="true">languageBox</string>
       </attribute>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="lang2">
       <property name="text">
        <string>g</string>
       </property>
       <attribute name="buttonGroup">
        <string notr="true">languageBox</string>
       </attribute>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>languageBox</sender>
   <signal>buttonClicked(int)</signal>
   <receiver>MacroControl</receiver>
   <slot>setMacroLanguage(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>54</x>
     <y>79</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>setMacroLanguage(int)</slot>
 </slots>
 <buttongroups>
  <buttongroup name="languageBox"/>
 </buttongroups>
</ui>
